bin/stg-build.sh

```
@@ -0,0 +1,117 @@
+fi
+
+wssScannerVersion=$(grep -Eo -m 1 'wss-scanner:([1-9][0-9]*)(\.[1-9][0-9]*)*(-[a-zA-Z0-9-]+)?' buildwithsast.sh | head -1)
+echo "Found version: $wssScannerVersion"
+if [ -z "$(docker images -q $wssScannerVersion 2> /dev/null)" ]; then
+  echo "wss-scanner:$ZIP_VERSION was not built successfully"
+  exit 1
+else
+  echo "$wssScannerVersion Built successfully!"
+fi
+
+wssScannerSastVersion=$(grep -Eo -m 1 'wss-scanner-sast:([1-9][0-9]*)(\.[1-9][0-9]*)*(-[a-zA-Z0-9-]+)?' buildwithsast.sh | head -1)
```

Shellcheck (0.11.0)

[warning] wssScannerSastVersion appears unused. Verify use (or export if used externally).

(SC2034)